clients/venice-thin-client/src/main/java/com/linkedin/venice/client/schema/RouterBackedSchemaReader.java

```diff
@@ -35,30 +35,30 @@
 public class RouterBackedSchemaReader implements SchemaReader {
   public static final String TYPE_KEY_SCHEMA = "key_schema";
   public static final String TYPE_VALUE_SCHEMA = "value_schema";
-  private static final ObjectMapper OBJECT_MAPPER = ObjectMapperFactory.getInstance();
-
-  // Ignore the unknown field while parsing the json response.
-  static {
-    OBJECT_MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Why this got removed?
I think this config is important to ignore the unknown fields, otherwise, json deserialization will fail.

Reply:
Because we're using internal flavor of `ObjectMapperFactory`, which already does it.

Reply:
Good to know, thanks!